src/style.css

```diff
@@ -1,3 +1,113 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could use 'font-family: Roboto, Arial, sans-serif;' to ensure there's a default font similar to Roboto if it fails to load.